### PR TITLE
doc: google - clean BQ async

### PR DESCRIPTION
**Pull request by raphaelauv**
clean the google provider doc referencing non existing class

### CODE REVIEW

File: docs/apache-airflow-providers-google/operators/cloud/bigquery.rst [hunk 1]
```diff
@@ -221,7 +220,7 @@ that row.
     :end-before: [END howto_operator_bigquery_get_data]
 
 The below example shows how to use
-:class:`~airflow.providers.google.cloud.operators.bigquery.BigQueryGetDataAsyncOperator`.
+:class:`~airflow.providers.google.cloud.operators.bigquery.BigQueryGetDataOperator`in async.
 Note that this is a deferrable operator which requires the Triggerer to be running on your Airflow
```
Comment (uranusjr) [inline docs/apache-airflow-providers-google/operators/cloud/bigquery.rst]:
```suggestion
:class:`~airflow.providers.google.cloud.operators.bigquery.BigQueryGetDataOperator`
in async (deferrable) mode. Note that a deferrable task requires the Triggerer to be
running on your Airflow
```

Same for other deferrable mentions below.

**Review by uranusjr — APPROVED** [review]

File: docs/apache-airflow-providers-google/operators/cloud/bigquery.rst [hunk 2]
```diff
@@ -221,9 +220,9 @@ that row.
     :end-before: [END howto_operator_bigquery_get_data]
 
 The below example shows how to use
-:class:`~airflow.providers.google.cloud.operators.bigquery.BigQueryGetDataAsyncOperator`.
-Note that this is a deferrable operator which requires the Triggerer to be running on your Airflow
-deployment.
+:class:`~airflow.providers.google.cloud.operators.bigquery.BigQueryGetDataOperator`
+in async (deferrable) mode. Note that a deferrable task requires the Triggerer to be
+running on your Airflow
```
Comment (jedcunningham) [inline docs/apache-airflow-providers-google/operators/cloud/bigquery.rst]:
```suggestion
running on your Airflow deployment.
```

Looks like that got accidentally dropped.

File: docs/apache-airflow-providers-google/operators/cloud/bigquery.rst [hunk 3]
```diff
@@ -320,9 +318,9 @@ with proper query job configuration that can be Jinja templated.
     :end-before: [END howto_operator_bigquery_insert_job]
 
 The below example shows how to use
-:class:`~airflow.providers.google.cloud.operators.bigquery.BigQueryInsertJobAsyncOperator`.
-Note that this is a deferrable operator which requires the Triggerer to be running on your Airflow
-deployment.
+:class:`~airflow.providers.google.cloud.operators.bigquery.BigQueryInsertJobOperator`
+in async (deferrable) mode. Note that a deferrable task requires the Triggerer to be
+running on your Airflow
```
Comment (jedcunningham) [inline docs/apache-airflow-providers-google/operators/cloud/bigquery.rst]:
```suggestion
running on your Airflow deployment.
```

**Review by jedcunningham — APPROVED** [review]

File: docs/apache-airflow-providers-google/operators/cloud/bigquery.rst (hunk 2 above)
Comment (raphaelauv) [reply docs/apache-airflow-providers-google/operators/cloud/bigquery.rst]: thanks

**Comment by raphaelauv** [comment]
could we merge ? thanks

**Pull request merged**